src/storage/store/column_chunk_data.cpp

```diff
@@ -207,8 +217,10 @@ void ColumnChunkData::updateStats(const common::ValueVector* vector,
             [this, vector, &selVector]<StorageValueType T>(T) {
```

Comment:
For the else branch here, can you take a look at `ValueVector::forEachNonNull` and see if that can be reused here? I think the logic is the same.

Reply:
updated